src/sites/image/imgpaying.com.js

```diff
@@ -53,7 +53,10 @@
 
   $.register({
     rule: {
-      host: /^img(rock|town)\.net$/,
+      host: [
+        /^imgtown\.net$/,
+        /^imgrock\.co$/,
```

Comment:
Since imgrock.co always redirect to imgrock.info, we probably only need to handle imgrock.info.